portuguese-brazilian/wca-guidelines.md

```diff
@@ -1,11 +1,11 @@
 # <wca-title>Orientações Adicionais da WCA
 
-<version>Versão: 1 de Janeiro de 2018
+<version>Versão: 1 de Agosto de 2020
 
 
 ## Notas
 
-Última revisão textual: 29 de Maio de 2018
+Última revisão textual: 1 de agosto de 2020
```

Comment:
Do you use capital letters for months in Portuguese? You have used them in the "Versão" but not in the "Última revisão textual".

Reply:
Thanks for spotting this! All of these should be in lower case. I'll make the necessary adjustments.